.sync.yml

```diff
@@ -44,23 +28,21 @@ Gemfile:
           - mswin
           - mingw
           - x64_mingw
-      - gem: bolt
-        version: '~> 1.15'
-        condition: ENV['GEM_BOLT']
+      - gem: 'puppet_litmus'
+        platforms:
+          - ruby
+          - mswin
+          - mingw
+          - x64_mingw
+        condition: "ENV['PUPPET_GEM_VERSION'].nil? or ENV['PUPPET_GEM_VERSION'] !~ %r{ 5}"
+    ':development':
+      - gem: puppet-lint-i18n
+      
 
 Rakefile:
   requires:
-    - puppet-lint/tasks/puppet-lint
-  extras:
-  extras: |
-    # The beaker task requires the test fixtures created by the spec_prep task
-    beaker_task_exists = Rake::Task.task_defined?('beaker')
-    spec_prep_task_exists = Rake::Task.task_defined?('spec_prep')
-    if beaker_task_exists && spec_prep_task_exists
-      beaker_task = Rake::Task['beaker']
-      spec_prep =  Rake::Task['spec_prep']
-      beaker_task.enhance(beaker_task.prerequisite_tasks << spec_prep)
-    end
+    - puppet_pot_generator/rake_tasks
```

Comment:
This could also be removed if `puppet-lint-i18n` is taken out.

Reply:
Thanks @eimlav removed the i18n section too